src/main/kotlin/org/jetbrains/kotlinx/jupyter/htmlUtil.kt

```diff
@@ -0,0 +1,76 @@
+package org.jetbrains.kotlinx.jupyter
+
+import org.jetbrains.kotlinx.jupyter.api.VariableState
+
+// TODO : perhaps, create factory class
+fun generateHTMLVarsReport(variablesMap: Map<String, VariableState>): String {
+    return buildString {
+        this.append(
+            """
+            <!DOCTYPE html>
+            <html>
```

Comment:
Do we need a full HTML document here, isn't simple `div` enough? Just wondering

Reply:
Moreover, we do not need div. It's generated by the default for the output HTML and builts table in